.github/workflows/cmake-ctest.yml

```diff
@@ -10,6 +10,7 @@ env:
   # Customize the CMake build type here (Release, Debug, RelWithDebInfo, etc.)
   BUILD_TYPE: Release
   WITH_DLT_UNIT_TESTS: ON
+  BUILD_GMOCK: OFF
```

Comment:
Hello @michael-methner 
With this flag we split gmock and only build gtest.
The submodule could be used directly.